docs/distributions/nixos/installation.md

```diff
@@ -185,15 +210,41 @@ If you would like to organize your configuration a little better, check out othe
 
 ## Wi-Fi and Bluetooth setup
 
+### Pre-Installation Steps
+
+Choose a method below and follow the [Wi-Fi and Bluetooth Guide on macOS](../../guides/wifi-bluetooth.md#on-macos), then come back.
+
+=== "Method 1"
```

Comment:
The macOS Recovery Image downloads will be method 4 in the wifi guide. You probably want to separate it out from Method 1, since it's a separate method.